engine/src/gldrv/winsys.cpp

```diff
@@ -215,14 +215,14 @@ static bool setup_sdl_video_mode(int *argc, char **argv) {
 
     int rs, gs, bs;
     rs = gs = bs = (bpp == 16) ? 5 : 8;
-    if (game_options()->rgb_pixel_format == "undefined") {
-        game_options()->rgb_pixel_format = ((bpp == 16) ? "555" : "888");
+    if (configuration()->graphics.rgb_pixel_format == "undefined") {
+        configuration()->graphics.rgb_pixel_format = ((bpp == 16) ? "555" : "888");
     }
-    if ((game_options()->rgb_pixel_format.length() == 3) && isdigit(game_options()->rgb_pixel_format[0])
-            && isdigit(game_options()->rgb_pixel_format[1]) && isdigit(game_options()->rgb_pixel_format[2])) {
-        rs = game_options()->rgb_pixel_format[0] - '0';
-        gs = game_options()->rgb_pixel_format[1] - '0';
-        bs = game_options()->rgb_pixel_format[2] - '0';
+    if ((configuration()->graphics.rgb_pixel_format.length() == 3) && isdigit(configuration()->graphics.rgb_pixel_format[0])
```

Comment:
I think we can deprecate this whole code. This isn't the 90's and we're not supporting 16 bit colors.

Reply:
Maybe later. I don't see that as a high priority right now.

Reply:
Opened #1240.